### PR TITLE
Updated xamarin sections to reflect the new SDK version

### PR DESCRIPTION
**Pull request by ziccardi**
See https://issues.jboss.org/browse/AEROGEAR-7743

Fixed few errors in the Android doc as well.

### CODE REVIEW

**Comment by ziccardi** [comment]
ping @finp @jstaffor

File: modules/ROOT/pages/_partials/attributes.adoc [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 :product-name: Mobile Services
 
-:release-number: 1.0.0
+:release-number: 2.0.0
```
Comment (finp) [inline modules/ROOT/pages/_partials/attributes.adoc]: @laurafitzgerald  proposed change to 2.0 ;)

Comment (laurafitzgerald) [reply modules/ROOT/pages/_partials/attributes.adoc]: this comment sounds like I proposed it :P The version bump is only for the SDK so from your comment above i believe this release version relates to the service not the SDK. correct?

Comment (finp) [reply modules/ROOT/pages/_partials/attributes.adoc]: @laurafitzgerald sorry, I did not mean you suggested it, but now that it has been suggested ;)

@ziccardi  AFAIK we're still at 1.0.0 wrt to this attribute (it's used throughout doc to represent the whole Mobile Services release)

**Comment by ziccardi** [comment]
@finp : Changed as requested. Mind giving another look?

**Pull request merged**